L10nSnapshotTests/L10nSnapshotTests.swift

```diff
@@ -199,6 +199,7 @@ class L10nSnapshotTests: L10nBaseSnapshotTests {
     func test20BookmarksTableContextMenu() {
         navigator.openURL(loremIpsumURL)
         navigator.performAction(Action.Bookmark)
+        navigator.createNewTab()
```

Comment:
This is a workaround. The navigator tries to long-press the tabs button and create a new tab, but instead 'New Private Tab' gets pressed incorrectly by the test runner. This method ensures the new tab is created and the app is on the home screen.

Reply:
@jhugman this is where I use a working navigation to the home screen (using createNewTab) to avoid screengraph having to long-press the tabs button and selecting 'New Tab'. The bug is that 'New Private Tab' gets tapped by the test runner incorrectly.